app/pipes/render-template.ts

```diff
@@ -1,9 +1,14 @@
 import {Pipe, PipeTransform} from '@angular/core';
 import * as nunjucks from 'nunjucks';
 
+let env = new nunjucks.Environment();
+env.addFilter('format_number', function(x: any) {
+  return x.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',');
```

Comment:
This may not work properly for a fraction numbers. I think, it's better to use [Number.toLocaleString()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString) or something like this.